django/contrib/gis/db/backends/mysql/operations.py

```diff
@@ -58,8 +58,8 @@ def gis_operators(self):
         }
         if self.connection.mysql_is_mariadb:
             operators["relate"] = SpatialOperator(func="ST_Relate")
-        if self.connection.mysql_is_mariadb and self.connection.mysql_version < (11, 7):
-            del operators["coveredby"]
+            if self.connection.mysql_version < (11, 7):
+                del operators["coveredby"]
```

Comment:
I think we can move this change into the previous commit 59657d1babbeee3d720f44eed8e93a3eaf5738ce

Reply:
True, will update when the first commit is merged :+1:

Reply:
Merged ef28b05767482bf9c98acc19ffd1334f75165b5a :rocket:

Reply:
Looks perfect thank you :star: